test/libyul/yulOptimizerTests/loopInvariantCodeMotion/move_memory_function.yul

```diff
@@ -0,0 +1,27 @@
+{
+  function f() -> x { x := g() }
```

Comment:
What is the point of these functions? Can you either modify this test or add another test that shows that the memory effects of user-defined functions are properly inferred? I.e. add a test where a local function performs an `mload` and show that it is moved out.

Reply:
I've added a `mload` and a `sload` now.